src/App.js

```diff
@@ -1,11 +1,15 @@
+import Body from './Parts/Body';
 import './App.css';
 import 'antd/dist/antd.css'; // or 'antd/dist/antd.less'
 import NavBar from './Parts/NavBar';
 
 function App() {
   return (
-    <div className='App'>
+    <div className="App">
       <NavBar />
+       <header className="App-header">
```

Comment:
This probably shouldn't be in a header, I don't know if we need this? We can remove this in a future ticket.